controlplane/cmd/nsmd/nsmd.go

```diff
@@ -14,11 +17,23 @@ import (
 func main() {
 	model := model.NewModel()
 
+	registryAddress := os.Getenv("K8S_REGISTRY_ADDRESS")
```

Comment:
On second look, I don't like this name. It is K8s specific when it should be NSM specific.